recipe/build.sh

```diff
@@ -18,7 +18,7 @@ mkdir -p $PREFIX/cuda/lib
 
 cp $RECIPE_DIR/LICENSE $PREFIX/doc/nvidia/cuDNN_LICENSE
 cp -r $SRC_DIR/cudnn/lib64/ $PREFIX/lib/
-cp -r $SRC_DIR/cudnn-dev/include/cudnn*.h $PREFIX/include/cudnn.h
+cp -r $SRC_DIR/cudnn-dev/include $PREFIX
```

Comment:
Is this going to be a problem? It looks like in the past we were converting a file name from cudnnSOMETHING.h to cudnn.h. Is CuDNN 8 setup differently?

Reply:
cudnn-dev7
```
./usr/include/cudnn_v7.h
./usr/lib64/libcudnn_static_v7.a
```
```
./usr/include/cudnn_adv_infer_v8.h
./usr/include/cudnn_adv_train_v8.h
./usr/include/cudnn_backend_v8.h
./usr/include/cudnn_cnn_infer_v8.h
./usr/include/cudnn_cnn_train_v8.h
./usr/include/cudnn_ops_infer_v8.h
./usr/include/cudnn_ops_train_v8.h
./usr/include/cudnn_v8.h
./usr/include/cudnn_version_v8.h
./usr/lib64/libcudnn_static_v8.a
```

So yea, perhaps a problem if we want to continue dropping that `_v` part.

Reply:
and I guess we'd want to drop the `_v8` on all of the .h files... I don't remember the reason why we currently do that for _v7.